test_fms/run_test.sh

```diff
@@ -0,0 +1,35 @@
+# Function to set up and run tests. 
+# Inputs: 
+# {1} Name of test 
+# {2} Number of processors 
+# {3} Set to skip, if the test is meant to fail 
```

Comment:
Usually in testing we mean that a test that is "meant to fail" is not one that is skipped, but a test that is run knowing it will fail, for example, because of bad or missing input.

It's not clear to my why there is so much effort to accommodate skipping tests. Instead, get your tests working! Building in tests that are just skipped is a bit strange. Why have tests if you are going to skip them?